tldw_Server_API/app/core/Ingestion_Media_Processing/persistence.py

```
@@ -1602,6 +1603,32 @@ def _resolve_media_add_embeddings_mode(form_data: Any) -> str:
     return "auto"
 
 
+def _mark_media_embeddings_complete(db: Any, media_id: int) -> None:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `db`

(ANN401)